### DISCUSSION TITLE
[Ideas] Add the possibility to play all artist's music on an artist page

### DISCUSSION THREAD

**Discussion by Multigriffes** [navidrome/navidrome#2898] [Ideas]
The goal is to add a button to queue artist's music like in other players
![Capture d'écran 2024-03-10 103204](https://github.com/navidrome/navidrome/assets/90653170/b8ed8205-7e2b-4e8e-a669-9d4260ea7105)

**Comment by metalheim** [comment]
yes, that would be cool. 
I have it in my issue that is tracking the "enhancing the artist page" progress: #961

**Reply by Multigriffes** [reply]
Cool, I haven't seen that.

**Reply by Multigriffes** [reply]
Should I close the discussion ?
